src/components/Nav/index.tsx

```diff
@@ -59,9 +59,6 @@ const InnerContent = styled.div`
   display: flex;
   justify-content: space-between;
   width: 100%;
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    display: none;
-  }
```

Comment:
Unfortunately, simply removing this will break the layout at smaller screens.
![Screen Shot 2023-03-04 at 13 23 18](https://user-images.githubusercontent.com/65234762/222922740-e7f65c04-9fa8-4de6-973c-05b7e6ca03bd.png)

A solution to apply for now would be to wrap `InnerContent` with the Chakra `Show` component in the render (line 395) and use the `useToken` hook to extract the correct breakpoint value.

So above the render return 
```tsx
// Make sure `useToken` is imported from Chakra
const lgBreakpoint = useToken("breakpoints", "lg")
```

And in the return at line 395

```tsx
<Show above={lgBreakpoint} >
  <InnerContent>{/*  */}</InnerContent>
</Show>
```

For @pettinarip, since I am handling migration of the entire `Nav`, I guess up to you on if you want to get this fix in now, since I am waiting on a couple of other PRs as well that involve the `Nav`

Reply:
@TylerAPfledderer this is what i have been waiting for, "The Feedback". 
thankyou, I will apply these changes.

Reply:
@TylerAPfledderer @pettinarip @rohitt-gupta I found a simpler way to do this just using Flex for now. I know we are planning on migrating this fully in the future, but for the purposes of this PR I think its a bit cleaner and achieves the same thing. Going to merge with that for now.

Reply:
Cool, thanks for the feedback @TylerAPfledderer.

Thanks @rohitt-gupta for the PR and agreed @corwintines, lets have this as a patch for now.